scenariorun/src/integrationTest/kotlin/com/cosmotech/scenariorun/service/ScenarioRunServiceRBACTest.kt

```diff
@@ -0,0 +1,4628 @@
+                      mutableListOf(datasetSaved.id!!))
+              val scenarioRunSaved =
+                  scenariorunApiService.runScenario(
+                      organizationSaved.id!!, workspaceSaved.id!!, scenarioSaved.id!!)
+              every { getCurrentAccountIdentifier(any()) } returns TEST_USER_MAIL
+
+              if (shouldThrow) {
+                val exception =
+                    assertThrows<CsmAccessForbiddenException> {
+                      scenariorunApiService.stopScenarioRun(
+                          organizationSaved.id!!, scenarioRunSaved.id!!)
+                    }
+                if (role == ROLE_USER || role == ROLE_NONE) {
+                  assertEquals(
+                      "RBAC ${scenarioSaved.id!!} - User does not have permission $PERMISSION_READ",
+                      exception.message)
+                } else {
+                  assertEquals(
+                      "RBAC ${scenarioSaved.id!!} - User does not have permission $PERMISSION_WRITE",
+                      exception.message)
+                }
+              } else {
+                assertDoesNotThrow {
+                  scenariorunApiService.stopScenarioRun(
+                      organizationSaved.id!!, scenarioRunSaved.id!!)
+                }
+              }
+            }
+          }
+
+  fun makeWorkspaceEventHubInfo(eventHubAvailable: Boolean): WorkspaceEventHubInfo {
+    return WorkspaceEventHubInfo(
+        eventHubNamespace = "eventHubNamespace",
+        eventHubAvailable = eventHubAvailable,
+        eventHubName = "eventHubName",
+        eventHubUri = "eventHubUri",
+        eventHubSasKeyName = "eventHubSasKeyName",
+        eventHubSasKey = "eventHubSasKey",
+        eventHubCredentialType =
+            CsmPlatformProperties.CsmPlatformAzure.CsmPlatformAzureEventBus.Authentication.Strategy
+                .SHARED_ACCESS_POLICY)
+  }
+
+  fun makeConnector(name: String = "connector"): Connector {
+    return Connector(
+        key = UUID.randomUUID().toString(),
+        name = name,
+        repository = "/repository",
+        version = "1.0",
+        ioTypes = listOf(Connector.IoTypes.read))
+  }
+
+  fun makeDataset(
+      organizationId: String,
+      connector: Connector,
+      userName: String = TEST_USER_MAIL,
+      role: String = ROLE_ADMIN
+  ): Dataset {
+    return Dataset(
+        name = "Dataset",
+        organizationId = organizationId,
+        ownerId = "ownerId",
+        connector =
+            DatasetConnector(
+                id = connector.id,
+                name = connector.name,
+                version = connector.version,
+            ),
+        security =
+            DatasetSecurity(
+                default = ROLE_NONE,
+                accessControlList =
+                    mutableListOf(
+                        DatasetAccessControl(id = CONNECTED_ADMIN_USER, role = ROLE_ADMIN),
+                        DatasetAccessControl(id = userName, role = role))))
```

Comment:
Standardize AccessControle definition for all make functions.
use "id =" and "role =" everywhere (or don't but everywhere)
define CONNECTED_ADMIN_USER first or second everywhere